docs/new-architecture-library-intro.md

```diff
@@ -173,6 +178,8 @@ Codegen can be configured in the `package.json` file of your Library. Add the fo
 - The `name` field is the name of the library.
 - The `type` field is used to identify the type of module we want to create. We suggest keeping `all` to support libraries that contain both Turbo Native Module and Fabric Native Components.
 - The `jsSrcsDir` is the directory where the codegen will start looking for JavaScript specs.
+- The `outputDir` this is the output directory for the generated code. Can be defined in a platform-independent way, like `"outputDir": "universal/generated"`, or per-platform as in the example above.
```

Comment:
```suggestion
- The `outputDir` field is the output directory for the generated code. It can be defined in a platform-independent way, like `"outputDir": "universal/generated"`, or per-platform as in the example above.
```